demoshell/main.py

```diff
@@ -137,6 +148,31 @@ def extend_text(self, style, text):
     def received_output(self, data, style):
         self.extend_text(style, data.decode('utf-8'))
 
+    def _load_config(self):
+        appname = "DemoShell"
+        appauthor = "Doug Hellman"    
```

Comment:
Let's leave the author part out. I want other people to feel like their contributions are important so I don't want my name here.